Source/Android/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
```

Comment:
I think the google maven repo might need to be in buildscript. The new form in gradle 4.1 is just "google()". Replace the maven { url... down below with google() as well.

Reply:
correct: 
https://developer.android.com/studio/build/dependencies.html#google-maven